ops/terraform/services/eft/lambda_src/sftp_outbound_transfer/tests/test_sftp_outbound_transfer.py

```diff
@@ -481,3 +486,50 @@ def test_it_executes_sftp_transfer_process_if_event_is_valid(self):
             )
             == 2
         )
+
+    @mock.patch(f"{MODULE_UNDER_TEST}.handle_sqs_message", mock_handle_sqs_message)
+    def test_poll_queue(self):
+        mock_sqs_client = mock.Mock()
+        mock_boto3_client.return_value = mock_sqs_client
+        mock_sqs_client.get_queue_url.return_value = {"QueueUrl": "mock_url"}
+        mock_sqs_client.receive_message.return_value = {
+            "Messages": [
+            {
+                "Body": json.dumps({"Message": json.dumps({"Records": [S3_MESSAGE]})}),
+                "ReceiptHandle": "mock_receipt_handle"
+
+            }
+            ]
+        }
+        poll_queue(False)
+        mock_handle_sqs_message.assert_called_once_with({"Records": [S3_MESSAGE]})
+        mock_sqs_client.get_queue_url.assert_called_once()
+        mock_sqs_client.receive_message.assert_called_once()
+        mock_sqs_client.delete_message.assert_called_once()
+
+    @mock.patch(f"{MODULE_UNDER_TEST}.handle_sqs_message", mock_handle_sqs_message)
+    def test_poll_queue_invalid_message(self):
+        mock_sqs_client = mock.Mock()
+        mock_boto3_client.return_value = mock_sqs_client
+        mock_sqs_client.get_queue_url.return_value = {"QueueUrl": "mock_url"}
+        mock_sqs_client.receive_message.return_value = []
+        poll_queue(False)
+        mock_handle_sqs_message.assert_not_called()
+
+        
+
+    @mock.patch(f"{MODULE_UNDER_TEST}._handle_s3_event", mock_handle_s3_event)
+    def test_handle_sqs_message(self):
+        handle_sqs_message({"Records": [S3_MESSAGE]})
+        mock_handle_s3_event.assert_called_once_with(s3_object_key=S3_MESSAGE["s3"]["object"]["key"])
+    
+    @mock.patch(f"{MODULE_UNDER_TEST}._handle_s3_event", mock_handle_s3_event)
+    def test_handle_sqs_message_invalid_Records(self):
+        with pytest.raises(KeyError) as exc_info:
+            handle_sqs_message({"Records": [{"somekey": "somevalue"}]})    
+            
+
+        
+
+        
+
```

Comment:
@dondevun I want to keep Mitch's spirit alive. Not sure if you've done this already but can you run the black formatter. He recommended using it https://black.readthedocs.io/en/stable/ to format Python code in BFD. 

pip install black
# from the directory of this file:
black --preview . --line-length 100